src/main/resources/lang/en_us/messages.yml

```
@@ -566,4 +563,8 @@ BTN_CONFIG_STATUS_UNSUCCESSFUL_HTTP_REQUEST: "Failed to send HTTP request to URL
 BTN_CONFIG_STATUS_UNSUCCESSFUL_INCOMPATIBLE_BTN_PROTOCOL_VERSION_CLIENT: "The current client BTN protocol implementation version is too old, local implementation version: {}, server requires at least: {}"
 BTN_CONFIG_STATUS_UNSUCCESSFUL_INCOMPATIBLE_BTN_PROTOCOL_VERSION_SERVER: "The current server BTN protocol implementation version is too old, server implementation version: {}, local requires at least: {}"
 BTN_CONFIG_STATUS_SUCCESSFUL: "Successfully connected to the BTN server"
-BTN_CONFIG_STATUS_EXCEPTION: "An unexpected error occurred while connecting to the BTN server: {}"
\ No newline at end of file
+BTN_CONFIG_STATUS_EXCEPTION: "An unexpected error occurred while connecting to the BTN server: {}"
+DOWNLOADER_QB_FAILED_REQUEST_TRACKER_LIST_ON_TORRENT: "Request tracker list on torrent {} failed：{} - {}"
+TRACKER_FAILED_TO_UPDATE_TRACKER: "Failed to update tracker list on torrent {}：{} - {}"
+UTILITIES_TRACKER_REPLACED: "Successfully replaced {} torrent's tracker"
+DOWNLOADER_BIGLYBT_INCORRECT_ADAPTER_VERSION: "Currently installed BiglyBT adapter version is NOT supported, please upgrade to  {} or higher."
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)